node/service/src/benchmarking.rs

```diff
@@ -130,10 +104,7 @@ impl frame_benchmarking_cli::ExtrinsicBuilder for RemarkBuilder {
 	}
 
 	fn build(&self, nonce: u32) -> std::result::Result<OpaqueExtrinsic, &'static str> {
-		let period = polkadot_runtime_common::BlockHashCount::get()
-			.checked_next_power_of_two()
-			.map(|c| c / 2)
-			.unwrap_or(2) as u64;
+		let period = 128;
```

Comment:
I understand why it was hardcoded, but a comment on how it's calculated would be nice ;)